.github/workflows/performance.yml

```
@@ -2,6 +2,10 @@ name: Performance
 
 on:
   pull_request:
+    paths:
+      - client/src/**
+      - .github/workflows/performance.yml
+      - yarn.lock
```

If the package.json `script` values change
```suggestion
      - package.json
      - yarn.lock
```

---

Fair enough. Better safe than sorry.